### DISCUSSION TITLE
Working files for the original

### DISCUSSION THREAD

**Discussion by BlazeDrake** [jacobfortune/Songs-Of-War-Addon#4] [General]
Hello! I'm glad to see that you've been working on the addon. I probably won't be working on it myself anymore, but I was able to update the original files to work with 1.20.30(as far as I was testing). I just wanted to let you know in case you wanted to take them to work off of, as I saw you mentioned in your comments that you weren't fully able to replicate them. If not, no worries at all. I also did add a link to this github in the addon description if that's alright. I can remove it as well if you'd prefer. Feel free to hit me up on discord as well if you have any questions

**Comment by lesleyespire** [comment]
Thankyou so much, I am planning to add jaggtahan species to this and i basically completed it, your addon has literally been incredible so thankyou again. I am Solos on discord by the way

**Comment by lesleyespire** [comment]
When you say working files to you mean that they are the entire addon working for 1.20.30 or just the original files? I managed to update but i could not get songs to work on the ardoni shrine structures

**Reply by BlazeDrake** [reply]
Just the files that I made, I haven't really checked out what you did specifically. feel free to copy paste the functions, or if you'd like I can walk you through how the system works. 

The basic idea is that the shrine center block runs a function on the player if they're an ardoni that tests for each kind of song at specific positions, and then gives them the activated item if the block is there
